### ISSUE TITLE
Add LICENSE?

### ISSUE THREAD

**Issue by johnpryan** [rxlabz/panache#40]
The license is currently not defined in the LICENSE file

**Comment by nav9** [comment]
I agree with John. Kindly put up a license file that explicitly states what license you are releasing the software with, and whether there are no restrictions on commercial use of what is generated by Panache.
